apps/mobile/src/components/TosFlow/index.tsx

```diff
@@ -0,0 +1,23 @@
+import {createNativeStackNavigator} from '@react-navigation/native-stack'
+import TosScreen from './components/TosScreen'
+import {type TosStackParamsList} from '../../navigationTypes'
+import FaqScreen from './components/FqaScreen'
+
+const TosStack = createNativeStackNavigator<TosStackParamsList>()
+
+function TosFlow(): JSX.Element {
+  return (
+    <TosStack.Navigator
```

Comment:
Move to root navigator. Make sure screens can be accessed even when user is not logged in.